example/web/src/components/Form.tsx

```diff
@@ -8,7 +9,7 @@ type FormData = {
 type Props<T extends FormData> = {
   name: string;
   onSubmit: React.FormEventHandler;
-  keys: (keyof T)[];
+  keys: formFields;
```

Comment:
Because this is not a string array any more, let's rename `keys` to `fieldsMetadata`.